exercises/concept/elyses-analytic-enchantments/.docs/instructions.md

```diff
@@ -0,0 +1,66 @@
+# Instructions
+
+Elyse, magician-to-be, continues her training.
+She will be given several stacks of cards that she needs to perform her tricks.
+To make things a bit easier, she only uses the cards 1 to 10.
+She only has one of each card, there are no duplicates in her stacks of cards.
+
+In this exercise, use methods from Julia's Base library to analyse the contents of an array.
+
+## 1. Determine if a card is present
+
+Elyse wants to determine if a card is present in the stack -- in other words, if the stack contains a specific number:
+
+```julia
+julia> card = 2;
+
+julia> find_card([9, 7, 3, 2], card)
+4
+```
+
+## 2. Find the position of a card
```

Comment:
The three `find` tasks should specify what to do if a target cannot be found or promise that the card will always be found.

I prefer that we tell students to return `nothing` or a result type.